Sources/_FoundationCShims/platform_shims.c

```diff
@@ -67,3 +67,18 @@ mach_port_t _platform_mach_task_self() {
 }
 #endif
 
+int _platform_shims_clock_monotonic_raw_gettime(struct timespec * _Nonnull ts) {
```

Comment:
Looks like the Windows failures are related:

```
C:\source\swift-foundation\Sources\_FoundationCShims\platform_shims.c:74:12: error: call to undeclared function 'clock_gettime'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   74 |     return clock_gettime(CLOCK_MONOTONIC_RAW, ts);
      |            ^
```

Do the implementation (and header) need to be guarded by a `__has_include` check?

Reply:
Since windows provides `time.h` header itself, we need to exclude them by `#if !defined(_WIN32)`